models/repo/repo.go

```diff
@@ -222,6 +222,30 @@ func (repo *Repository) MustOwner() *user_model.User {
 	return repo.mustOwner(db.DefaultContext)
 }
 
+// LoadAttributes loads attributes of the repository.
+func (repo *Repository) LoadAttributes(ctx context.Context) error {
+	// Load owner
+	if err := repo.GetOwner(ctx); err != nil {
+		return fmt.Errorf("load owner: %w", err)
+	}
+
+	// Load primary language
+	stats := make(LanguageStatList, 0, 1)
```

Comment:
Is it possible to avoid using a slice? Given it's only 1 entry.

Reply:
That would require introducing duplicate code for `LanguageStat` attributes with no real gain. (`LanguageStatList` has `LoadAttributes` while `LanguageStat` does not have)